### PR TITLE
[doc][ybm] Enable PingOne fed auth

### PR DESCRIPTION
**Pull request by ddhodge**
Enable PingOne fed auth

@netlify /preview/yugabyte-cloud/managed-security/managed-authentication/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 1]
```diff
@@ -184,4 +136,50 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+
+  {{% tab header="Okta" lang="okta" %}}
+
+**Create an application in Okta**
+
+To use Okta for your IdP, do the following:
+
+1. Sign in to your Okta account and create an app integration.
+
+    - In the Admin Console, go to **Applications > Applications**, and click **Create App Integration**.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:148]: Applications > Applications?

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 2]
```diff
@@ -184,4 +136,50 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+- Client ID and secret of the application you created. These are provided on the **General** tab.
+- Your Okta domain. Click your account name in the top right corner of the Okta Admin Console; the domain is displayed under your account name.
+
+For more information, refer to [App integrations](https://help.okta.com/oie/en-us/content/topics/apps/apps_apps.htm) in the Okta Identity Engine documentation.
+
+**Configure**
+
+To configure federated authentication in YugabyteDB Managed, do the following:
+
+1. Navigate to **Security > Access Control > Authentication** and click **Enable Federated Authentication** to display the **Enable Federated Authentication** dialog.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:175]:
```suggestion
1. Navigate to **Security > Access Control > Authentication**, and click **Enable Federated Authentication** to display the **Enable Federated Authentication** dialog.
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 3]
```diff
@@ -184,4 +136,50 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+1. Choose Okta identity provider.
+1. Enter the client ID and secret of the Okta application you created.
+1. Enter the Okta domain for your application.
+1. Click **Enable**.
+
+You are redirected to sign in to your IdP to test the connection. Once test connection is successful, federated authentication is enabled.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:181]:
```suggestion
You are redirected to sign in to your IdP to test the connection. After the test connection is successful, federated authentication is enabled.
```

**Review by aishwarya24 — APPROVED** [review]
some comments

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 4]
```diff
@@ -184,4 +136,50 @@ To configure federated authentication in YugabyteDB Managed, do the following:
 You are redirected to sign in to your IdP to test the connection. Once test connection is successful, federated authentication is enabled.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:136]:
```suggestion
You are redirected to sign in to your IdP to test the connection. After the test connection is successful, federated authentication is enabled.
```

**Pull request merged**